.github/workflows/config.yml

```diff
@@ -349,20 +349,26 @@ jobs:
           path: test/tests_yml_files
           name: precomputed_buildstocks
 
-      - name: Build documentation
+      - name: Build technical development guide
         run: |
           sudo gem install oga
-          ruby docs/read_the_docs/source/workflow_inputs/create_characteristics_rst.rb
-          ruby docs/read_the_docs/source/workflow_outputs/csv_tables.rb
-          cd docs/read_the_docs
+          ruby docs/technical_development_guide/source/workflow_inputs/create_characteristics_rst.rb
+          ruby docs/technical_development_guide/source/workflow_outputs/csv_tables.rb
+          cd docs/technical_development_guide
           pip install changelog
           make html SPHINXOPTS="-W --keep-going -n"
-
-      - name: Save Docs
+      
+      - name: Save technical development guide
```

Comment:
TDG gets built and uploaded as an artifact; it isn't committed to the repository.
TRG gets built and committed to the repository; it isn't uploaded as an artifact.

Should we be consistent here?

Reply:
Yeah. I wanted the PDF initially, but the compiled TRG is 15MB. Uploading the TRG as an artifact: [6182fe9](https://github.com/NREL/resstock/pull/1338/commits/6182fe9eea5918bff517e4f4012b8b0b49d94266). Deleting the PDF from committed files: [e20406b](https://github.com/NREL/resstock/pull/1338/commits/e20406b81cdb6acfdf5b23dd5270b890a4fa110a)